autoprotocol/instruction.py

```diff
@@ -174,14 +177,14 @@ def get_wells(self, op_data):
             all source and destination wells for instructions such as `liquid_handle`.
         """
         all_wells = []
-        if type(op_data) is dict:
-            for k, v in op_data.items():
-                all_wells.append(self.get_wells(v))
-        elif type(op_data) is list:
+        if isinstance(op_data, dict):
+            for j, k in op_data.items():
```

Comment:
Would be good to know what j and k are here

Reply:
this...I do not know 😅 again, pylint was not happy with the original variables of a piece of code that was written before me. I'll look into what they are and change it!